src/main/kotlin/domain/Cell.kt

```diff
@@ -0,0 +1,22 @@
+package domain
+
+data class Cell private constructor(private var cellState: CellState, private var mineNumber: MineNumber) {
+    constructor() : this(CellState(), MineNumber())
```

Comment:
```suggestion
data class Cell private constructor(
    private var cellState: CellState = CellState(),
    private var mineNumber: MineNumber = MineNumber()
) {
```
디폴트값을 활용하시면 좀 더 코틀린스럽게 작성할 수 있어 보여요

Reply:
data 클래스는 copy 함수를 지원하기 때문에, private constructor는 의미가 없어 보여요
private를 제거해주시면 좋을 것 같네요

Reply:
오 과연 그렇네요. data class 가 아직 익숙치 않아서 자바스러운 생성자 선언이 되어버렸는데,
이렇게 바꾸니깐 훨씬 깔끔하네요.